device-types/Sophos/XGS-3100.yaml

```diff
@@ -0,0 +1,55 @@
+---
+manufacturer: Sophos
+model: XGS 3100
+slug: sophos-xgs-3100
+u_height: 1
+is_full_depth: false
+subdevice_role: parent
+airflow: front-to-rear
+console-server-ports:
```

Comment:
Are these interfaces for console access to the device, or does the device act as a server for these console ports which would thus connect to other devices console ports? If it is the former, this should be defined as `console-ports` rather than `console-server-ports`

Reply:
Thanks for sanity checking, you are right - my error. Attribute is corrected now.